.github/workflows/pull-release.yml

```diff
@@ -0,0 +1,137 @@
+name: get latest helm charts
+on:
+  schedule:
+    - cron: '00 06 * * 5' #every friday at 11:30 AM IST/ 6 AM UST
```

Comment:
@rish691 , this is logic to run the job at scheduled time. 

check this: https://crontab.guru/#00_06_*_*_5